src/manifestoo/main.py

```diff
@@ -43,6 +45,25 @@ def callback(
         ),
         show_default=False,
     ),
+    select_git_modified: str = typer.Option(
+        None,
+        "--select-git-modified",
```

Comment:
Or `--select-vcs-modified` ? To open the door to supporting other VCSes without changing the UI. Not sure.

Reply:
I asked myself the same question. If you want to support other VCS, you need a way to specify which one, to implement it, and someone to support the feature. I find it better not to suggest it works generically if only git is supported.